### DISCUSSION TITLE
Allow Difficulties created by user to not be deleted when map is updated

### DISCUSSION THREAD

**Discussion by Lyti4n** [ppy/osu#27152] [Ideas]
This could a setting or something, as of right now if i make a new difficulty on a mapset and it then gets updated I cant update the map without deleting that difficulty. This is a deal breaker for me since i frequently map by editing someone else difficulty cause I'm too lazy to do the whole process, which are frequently using unranked maps so get updated and I would hate to lose those difficulties.

**Comment by Rekunan** [comment]
I think it should also be possible, or as an alternative to this, to be able to split off any selected difficulties into a new separate set.
